packages/x-date-pickers-pro/src/internals/hooks/models/useRangePicker.ts

```diff
@@ -1,3 +1,4 @@
+'use client';
```

Comment:
Sure.
If a top-level entry point consuming such hooks uses `use client` already, then anything below it is inherently a client-only component.
These extra lines won't hurt, I just raised a point for discussion, because the approach to code structure differs between the `mui-material` repo and ours. 🙈

Reply:
Solved by https://github.com/mui/mui-x/pull/14562#discussion_r1752181658
We can add `@ignore` to the hooks we don't want to add "use client" to

Reply:
Nice. 👍